### ISSUE TITLE
Fullscreen mode: Do not use abbreviations in the description

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#57516]
### Description

In the Post editor > Options > Fullscreen mode, the setting description uses the English abbreviation `UI`.

https://github.com/WordPress/gutenberg/blob/0b522a9e69f585098811b505a7bb666938c1dc3d/packages/edit-post/src/components/header/writing-menu/index.js#L81

As a general rule, abbreviations should be expanded on first use. However, iin this specific case, as a sued I'm not supposed to know what an 'UI' is. That's a pretty technical, design-related jargon. I'm not sure using such abbreviations in an _user interface_ makes the user experience any better.

For accessibility, language should always be as simple and understandable as possible. 'UI' adds cognitive load and should be replaces by full text.

Also, all translations I checked expand this abbreviation to 'user interface' or similar, which is a clear sign that 'UI' doesn't help clarity.

Screenshots:

![Screenshot 2024-01-03 at 14 40 37](https://github.com/WordPress/gutenberg/assets/1682452/79f733ce-b3e4-42b9-9707-f05b8255edc9)

![Screenshot 2024-01-03 at 12 46 49](https://github.com/WordPress/gutenberg/assets/1682452/67e35ca9-362b-4dcb-807d-4233e412a896)

![Screenshot 2024-01-03 at 12 51 43](https://github.com/WordPress/gutenberg/assets/1682452/54c1fd57-d19a-4ec1-a0f9-7051ffd53fc7)

![Screenshot 2024-01-03 at 12 44 17](https://github.com/WordPress/gutenberg/assets/1682452/ff318353-8936-4ba2-b139-b21a04e3b605)

![Screenshot 2024-01-03 at 12 57 01](https://github.com/WordPress/gutenberg/assets/1682452/ebfa0989-32b3-4306-b8a1-a6223498d00a)

![Screenshot 2024-01-03 at 12 54 18](https://github.com/WordPress/gutenberg/assets/1682452/2b30549b-3679-4d45-a686-46a2f48adee4)

![Screenshot 2024-01-03 at 12 55 35](https://github.com/WordPress/gutenberg/assets/1682452/9605be3f-2643-43fc-872b-8c00e2893e90)

Introduced in https://github.com/WordPress/gutenberg/commit/1e2d8b0df1147b144aea91406e3a71740f3785e9

### Step-by-step reproduction instructions

See screenshots.

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes
